components/raftstore-v2/src/operation/mod.rs

```diff
@@ -8,3 +8,5 @@ mod ready;
 pub use command::{SimpleWriteDecoder, SimpleWriteEncoder};
 pub use life::DestroyProgress;
 pub use ready::AsyncWriter;
+
+pub(crate) use self::query::LocalReader;
```

Comment:
If it could be used by other mods, using `pub` would be better.

Reply:
It's supposed not to be used by other module.

Reply:
In v1's code, it's called in components/server as well.